.gitlab-ci-check-commits.yml

```diff
@@ -20,4 +20,4 @@ test:check-commits:
     - git clone http://github.com/mendersoftware/mendertesting /tmp/mendertesting
   script:
     # Check commit compliance.
-    - /tmp/mendertesting/check_commits.sh
+    - /tmp/mendertesting/check_commits.sh ${MENDERTESTING_ARGS}
```

Comment:
How is this variable supposed to be used? Will that actually work across pipelines? I think the best would be to create two new templates: `.gitlab-ci-check-commits-signoffs.yml` and `...-changelogs.yml`